db/migrate/20230329184329_create_post.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+
```

Comment:
- [ ] I don't see any migration where you add the foreign key that relates posts table and users table. You should use migrations to add columns, and foreign keys, create tables, and alter tables. Avoid directly editing the schema file. Kindly add a migration to create a foreign key that relates the users table and the posts.